GitCommands/Git/GitModule.cs

```diff
@@ -3766,29 +3766,25 @@ public IList<GitItemStatus> GetCombinedDiffFileList(string shaOfMergeCommit)
         {
             var fileList = RunGitCmd("diff-tree --name-only -z --cc --no-commit-id " + shaOfMergeCommit);
 
-            var ret = new List<GitItemStatus>();
             if (string.IsNullOrWhiteSpace(fileList))
             {
-                return ret;
+                return new GitItemStatus[0];
```

Comment:
Pure curiosity, is `Enumerable.Empty<GitStatus>()` worse? 
Looking at it in the reflector, it creates a singleton instance of `GitItemStatus[0]`....

Reply:
No, this should have been changed to `Enumerable.Empty<GitItemStatus>()`. I'll push a fix shortly. Thanks.

And yes `Enumerable.Empty<>` returns a singleton empty array (as does `Array.Empty<>`) so assuming you use the empty array for a type more than once you can reduce allocations.

Reply:
Fixed. Needed to be `Array.Empty<>` here as the return type is `IList<>`.

Once these `GitModule` PRs are finalised I'll go through and change a bunch of these `IList<T>` and `T[]` return types to `IReadOnlyList<>`. Maybe some of the `IEnumerable<T>` as well, in cases where the collection is already materialised.